packages/console/src/components/ConnectorTester/index.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
   connectorFactoryId: string;
   connectorType: Exclude<ConnectorType, ConnectorType.Social>;
   className?: string;
+  validateConfigForm: () => Promise<boolean>;
```

Comment:
Try `useFormContext` to get `trigger` function, so that we can avoid having this new prop

Reply:
Good suggestion, updated.